invertedai/utils.py

```diff
@@ -713,7 +718,17 @@ def plot_frame(self, idx, ax=None, numbers=False, direction_vec=False,
 
     def animate_scene(self, output_name=None, start_idx=0, end_idx=-1, ax=None,
```

Comment:
> 

As I am going through the attributes and methods making docstrings, I have two questions as of now:
1. What does open_drive do? It appears to be a flag that enables different behaviour but what is the high level function of these different execution paths? 
2. It seems like there are a lot of parameters from the LocationResponse dataclass are input separately into the class. My question is why is this done separately instead of just inputting the LocationResponse directly?  Is there any use case where a user needs to modify these parameters after getting the LocationResponse individually since it is opening this tool up to misuse and seems overall clunky.

Reply:
1. I don't know what `open_drive` does unfortunately. Maybe that's something Alireza added?

2. If you can refactor that I would say go for it, seems like a sensible thing to do!

Reply:
This was said in-person but for documentation purposes, I will be taking this feedback and doing a larger refactor of ScenePlotter.